src/flamenco/snapshot/fd_snapshot_restore.h

```diff
@@ -81,8 +98,10 @@ fd_snapshot_restore_new( void *                               mem,
                          fd_acc_mgr_t *                       acc_mgr,
                          fd_funk_txn_t *                      txn,
                          fd_valloc_t                          valloc,
-                         void *                               cb_ctx,
-                         fd_snapshot_restore_cb_manifest_fn_t cb );
+                         void *                               cb_manifest_ctx,
+                         fd_snapshot_restore_cb_manifest_fn_t cb_manifest,
+                         void *                               cb_status_cache_ctx,
+                         fd_snapshot_restore_cb_status_cache_fn_t cb_status_cache );
```

Comment:
The callback context would typically be the same
```suggestion
                         void *                               cb_ctx,
                         fd_snapshot_restore_cb_manifest_fn_t cb_manifest,
                         fd_snapshot_restore_cb_status_cache_fn_t cb_status_cache );
```